### PR TITLE
Story/2419

### PR DESCRIPTION
**Pull request by kdwyer**

### CODE REVIEW

File: addons/udes_suggest_location/tests/test_stock_move_line.py [hunk 1]
```diff
@@ -234,3 +236,276 @@ def test_set_invalid_dest_location(self):
...
+##        # Mock functions, for suggest locations and validate
+##        # Suggest locations return empty locations
+##        cls.mock_suggested_locations = patch.object(
+##            cls.MoveLine.__class__,
+##            "suggest_locations",
+##            return_value=cls.test_check_location_04,
+##        )
+##        cls.mock_validate_location_dest = patch.object(
+##            cls.MoveLine.__class__, "validate_location_dest", return_value=None
+##        )
```
Comment (adamrpatrick) [inline addons/udes_suggest_location/tests/test_stock_move_line.py:317]: Still need these?

File: addons/udes_suggest_location/tests/test_stock_move_line.py [hunk 2]
```diff
@@ -234,3 +236,276 @@ def test_set_invalid_dest_location(self):
...
+
+        # Complete pickings 1 & 3. Picking 2 should have Check 01 suggested, as
+        # it shares an origin with Picking 1.
+        for picking, location in zip((cls.picking1, cls.picking3), (cls.test_check_location_01, cls.test_check_location_02)):
+            picking.move_line_ids.location_dest_id = location
+            picking.move_line_ids.qty_done = picking.move_line_ids.product_uom_qty
+            picking._action_done()
+
+        # Destination locations for each product
+##        cls.pick_apple_locs = cls.test_check_location_01 | cls.test_check_location_03
```
Comment (adamrpatrick) [inline addons/udes_suggest_location/tests/test_stock_move_line.py:298]: Still need this?

**Review by adamrpatrick — APPROVED** [review]

**Review by PeterAlabaster — APPROVED** [review]

File: addons/udes_suggest_location/tests/test_stock_move_line.py (hunk 1 above)
Comment (kdwyer) [reply addons/udes_suggest_location/tests/test_stock_move_line.py:317]: I don't know yet - that will be part of the tests clean up.

File: addons/udes_suggest_location/tests/test_stock_move_line.py (hunk 2 above)
Comment (kdwyer) [reply addons/udes_suggest_location/tests/test_stock_move_line.py]: Removed by 70ad9cce, please review.

**Pull request merged**